assembl/static2/js/app/components/cookies/cookieSetter.jsx

```diff
@@ -0,0 +1,72 @@
+// @flow
+/* global globalAnalytics */
+import React from 'react';
+import { I18n } from 'react-redux-i18n';
+import classnames from 'classnames';
+
+// Components
+import Helper from '../common/helper';
+import CookieSwitch from './cookieSwitch';
+
+import { COOKIE_TRANSLATION_KEYS } from '../../constants';
+
+export type CookieObject = {
+  name: string,
+  category: string,
+  accepted: boolean,
+  cookieType: string
+};
+
+type cookieSetterProps = {
+  handleToggle: Function,
+  cookie: CookieObject,
+  toggleCookieType: Function,
+  locale: string
+};
+
+// @$FlowFixMe globalAnalytics is a global variable
```

Comment:
you should probably declare the global variable in a new `flow/globals.flow.js` file (see this answer: https://stackoverflow.com/a/44376914/2318507 )

Reply:
Little issue: this variable is empty on localhost so I can't log its content. @vincentfretin globalAnalytics.host is a string and globalAnalytics.isActive is a boolean, right?

Reply:
globalAnalytics.piwik.isActive is a boolean yes, globalAnalytics.piwik.host is a string